daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -96,7 +88,9 @@ def _get_map_of_algorithms():
         'fin_check':                [[(validation, '_assert_all_finite', _daal_assert_all_finite), None]],
         'tsne':                     [[(manifold_module, 'TSNE', TSNE_daal4py), None]],
     }
-    if daal_check_version((2021,'P', 100)):
+    if LooseVersion(sklearn_version) < LooseVersion("0.22"):
```

Comment:
Does it make sense to support version 0.21? It's very hard?

Reply:
I don't think that performance improve for old versions is critical for us

Reply:
If merging the files 0.21 and 0.22 is not difficult, I would do it.